etc/rc.local.linux

```diff
@@ -1,24 +1,30 @@
 #!/bin/sh -e
 #
-# /etc/rc.local
-#
-# This script is executed at the end of each multiuser runlevel.
-# This means that once Krun has rebooted the machine, you will not
-# see a login prompt until all benchmarks have finished.
-#
-# Verify this script by running:
-#    sudo service rc.local start
-#
```

Comment:
I think we need to keep at least some of this comment (e.g. where the file should go, how to verify the script).